qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -444,10 +444,316 @@ def particle_number(orbitals, mapping="jordan_wigner", wires=None):
+    orbitals :math:`\phi_\alpha(r)`. Since the operator :math:`\hat{t}` acts only on the
+    spatial coordinates, the spin quantum numbers are indicated explicitly with the up/down arrows.
+    The operators :math:`\hat{c}^\dagger` and :math:`\hat{c}` are the particle creation
+    and annihilation operators, respectively, and
+    :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle` denotes the matrix elements of
+    the operator :math:`\hat{t}`
+
+    .. math::
+
+        \langle \alpha \vert \hat{t} \vert \beta \rangle = \int dr ~ \phi_\alpha^*(r)
+        \hat{t}(r) \phi_\beta(r).
+
+    If an active space is defined (see :func:`~.active_space`), the summation indices
+    run over the active orbitals and the contribution due to core orbitals is computed as
+    :math:`T_\mathrm{core} = 2 \sum_{\alpha\in \mathrm{core}}
+    \langle \alpha \vert \hat{t} \vert \alpha \rangle`.
+
+    Args:
+        matrix_elements (array[float]): 2D NumPy array with the matrix elements
+            :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle`
+        core (list): indices of core orbitals, i.e., the orbitals that are
+            not correlated in the many-body wave function
+        active (list): indices of active orbitals, i.e., the orbitals used to
+            build the correlated many-body wave function
+        cutoff (float): Cutoff value for including matrix elements. The
+            matrix elements with absolute value less than ``cutoff`` are neglected.
+
+    Returns:
+        tuple: Table of indices and matrix elements of the one-particle operator
+        and the contribution due to core orbitals. The returned table is a 2D Numpy
+        array where each row contains three elements, the *spin*-orbital indices
+        :math:`\alpha, \beta` and the matrix element
+        :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle`.
+
+    **Example**
+
+    >>> matrix_elements = np.array([[-4.72739313e+00, -1.05499666e-01, -1.66961416e-01,
+    ...                               6.18014041e-16,  2.86964662e-16, -3.46772026e-02],
+    ...                             [-1.05499666e-01, -1.49264622e+00, 3.28928073e-02,
+    ...                              -2.20398308e-16,  1.93277291e-16, 5.27078882e-02],
+    ...                             [-1.66961416e-01,  3.28928073e-02, -1.12554473e+00,
+    ...                              -2.82912389e-17,  2.55224784e-16, -3.04455743e-02],
+    ...                             [ 6.18014041e-16, -2.20398308e-16, -2.82912389e-17,
+    ...                              -1.13579985e+00, -1.94289029e-16, -2.36158697e-16],
+    ...                             [ 2.86964662e-16,  1.93277291e-16,  2.55224784e-16,
+    ...                              -2.77555756e-16, -1.13579985e+00,  2.06665432e-16],
+    ...                             [-3.46772026e-02,  5.27078882e-02, -3.04455743e-02,
+    ...                              -2.36158697e-16,  2.06665432e-16, -9.50966595e-01]]
```

Comment:
Note, this is missing the final `)`.

Also (and very minor), is it worth making it less spaced to the right?

```suggestion
    >>> matrix_elements = np.array(
    ...     [[-4.72739313e+00, -1.05499666e-01, -1.66961416e-01,
    ...       6.18014041e-16,  2.86964662e-16, -3.46772026e-02],
    ...      [-1.05499666e-01, -1.49264622e+00, 3.28928073e-02,
    ...       -2.20398308e-16,  1.93277291e-16, 5.27078882e-02],
    ...      [-1.66961416e-01,  3.28928073e-02, -1.12554473e+00,
    ...       -2.82912389e-17,  2.55224784e-16, -3.04455743e-02],
    ...      [ 6.18014041e-16, -2.20398308e-16, -2.82912389e-17,
    ...       -1.13579985e+00, -1.94289029e-16, -2.36158697e-16],
    ...      [ 2.86964662e-16,  1.93277291e-16,  2.55224784e-16,
    ...       -2.77555756e-16, -1.13579985e+00,  2.06665432e-16],
    ...      [-3.46772026e-02,  5.27078882e-02, -3.04455743e-02,
    ...       -2.36158697e-16,  2.06665432e-16, -9.50966595e-01]]
    ... )
```